lib/psm.cc

```diff
@@ -150,6 +150,7 @@ rpmRC rpmInstallSourcePackage(rpmts ts, FD_t fd,
     Header h = NULL;
     rpmpsm psm = NULL;
     rpmte te = NULL;
+    rpmPlugins plugins = NULL;
```

Comment:
I'd call this "origplugins" or something like that to make its purpose clearer.

Reply:
Indeed, fixed now.